### DISCUSSION TITLE
Add labels

### DISCUSSION THREAD

**Discussion by ManCaveMedia** [bootsie123/F1-Web-Viewer#22] [Ideas]
Would it be an idea to be able to add labels to the grid?
If you add a lot of driver video's, it fwould be nice to see which is which. 
(Some cars have the same color... 😉 )

**Comment by bootsie123** [comment]
That's actually a pretty good idea! I'm thinking of showing the team color and driver name on hover of the video. Thoughts on that? Alternatively the could be a toggle to hide and show it in the settings

**Reply by ManCaveMedia** [reply]
Would be cool to use the 3 letter abbreviation and the team color.

**Reply by x-f** [reply]
I support this idea with both hands. Driver cams are usually smaller that the main feeds and I wouldn't want to memorise colors of every helmet. :)

On hover - great. I wouldn't display the name all the time, but as an option some users probably would.
I have no opinion on necessity for team's color. 
Full name is probably easier than the abbreviation, if it is not already in the API data.
